### DISCUSSION TITLE
ssh timeout to github.com

### DISCUSSION THREAD

**Discussion by TauqeerAhmad5201** [pop-os/pop#2855] [General]
My SSH command `ssh -T git@github.com` is not working, says connection timed-out. However, the time I change to other network, it works or I change the OS, even though it works. 
Only occurring for Pop-OS on my home network. It is solved if I change either of the given parameters.

**Comment by jacobgkau** [comment]
Pop Shell (this repository) is for the window tiling GNOME extension. This issue does not sound related to Pop Shell.

If you disable Pop Shell using the Extensions app (green icon), does the issue still occur?

**Reply by TauqeerAhmad5201** [reply]
Hey @jacobgkau , thanks for guiding. Though the problem persists.

**Reply by jacobgkau** [reply]
Then this issue is not related to pop-shell. I will transfer your discussion to a more appropriate repository.
